system/kube-system/charts/ingress/templates/ingress-service.yaml

```diff
@@ -2,8 +2,10 @@ kind: Service
 apiVersion: v1
 
 metadata:
-  name: ingress-controller 
-
+  name: ingress-controller
+  annotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "9101"
```

Comment:
Why this port? Looking at the sources `/metrics` should be on the same port as `/healthz` which is `10254`.